source/includes/language-compatibility-table-python.rst

```diff
@@ -342,6 +360,9 @@ Python 2 Compatibility
 
 .. [#python-2-compat] PyMongo 4.0 and later is not compatible with Python 2
 
+.. [#ssl-4.0-issue] Python 3.10 and later are not compatible with TLS/SSL for
```

Comment:
This says "...and later **are** not compatible," while above it says "... and later **is** not compatible." I'm not sure which is correct but I think they should be the same

Reply:
Good catch. I think the first is not grammatical since it refers to multiple items. I'll fix that.